Shopify/shopify-app-template-remix
# Make it easier to get started with webhooks

### WHY are these changes introduced?
We want to make it easier to get started with declarative webhooks

### WHAT is this pull request doing?

1. Provide the config for mandatory webhook topics
2. Add a product create webhook, so partners can see a webhook trigger when pressing "generate a product"
3. Separate each webhook topic into it's own URL

### Test this PR

```bash
shopify app init --template=https://github.com/Shopify/shopify-app-template-remix#declarative-webhooks
```

1. Run `npm run dev`.  Do you see a message like this: `webhooks │ APP_UNINSTALLED webhook delivered`
2. Click `generate a product` in the UI.  Does the product create webhook trigger?
3. Does the webhook config match the routes correctly?
4. Does the code in each webhook make sense?

### Checklist

- [ ] I have made changes to the `README.md` file and other related documentation, if applicable
- [ ] I'm aware I need to create a new release when this PR is merged
